### DISCUSSION TITLE
Orca doesn't show my printers until I restart.

### DISCUSSION THREAD

**Discussion by akoolenbourke** [OrcaSlicer/OrcaSlicer#6034] [General]
Anyone else having this issue? When is start up Orca and make a new project or load a recent one, almost always I will somehow to have my actual 3 printer settings loaded. All the defaults load but my custom ones don't. If I then quit Orca and restart it they come back. It's stupid because I have to run Orca twice every time.

**Comment by smogan** [comment]
same here. since a while (dont know since when exactliy) my custom printer only shows up every other start of orcaslicer.

**Reply by akoolenbourke** [reply]
Have you ever built Orcaslicer yourself? I have and I wondered if something went weird that caused that. Given this and the laggy UI issue I have actually moved to BambuStudio for my P1S. Hopefully Orca will improve because I liked some of its features like per-filament PA.

**Reply by smogan** [reply]
No. I am just a user with an unsupported printer (Anycubix Kobra 2 Pro). I use the regular download version of orcaslicer (2.1.1 at the moment) for windows.
